src/icefabric/schemas/iceberg_tables/hydrofabric.py

```diff
@@ -194,7 +197,8 @@ def schema(cls) -> Schema:
             NestedField(38, "circ_mean.aspect", DoubleType(), required=False),
             NestedField(39, "dist_4.twi", StringType(), required=False),
             NestedField(40, "vpuid", StringType(), required=True),
-            identifier_field_ids=[1, 40],
+            NestedField(41, "glacier_percent", DoubleType(), required=False),
+            identifier_field_ids=[1, 41],
```

Comment:
```suggestion
            identifier_field_ids=[1, 40],
```
this should still be 1 and 40. This means what fields are required and can be used for partitioning. Since glacier percent isn't available everywhere yet I don't think this should be used as an identifier field

Reply:
fixed